keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -234,21 +234,17 @@ data class ResourceDeltaResolved(
   override val kind: String,
   override val id: String,
   override val application: String,
-  override val timestamp: Instant,
-  val desired: Any,
-  val current: Any
```

Comment:
No longer need the `desired` and `current` models in this event class since I'm not using it for artifact promotion any longer. It was really bloating the event history output as well.